src/window.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class WindowService {
+
+  get nativeWindow(): any {
+    return _window();
+  }
+}
+
+function _window(): any {
+  // return the global native browser window object
+  return typeof window !== 'undefined' ? window : undefined;
+}
```

Comment:
We should be able to replace the whole service with the following function, any thoughts?
```js
export function getNativeWindow() {
    return typeof window !== 'undefined' ? window : undefined;
}
```